storage/src/vespa/storage/visiting/commandqueue.h

```diff
@@ -141,11 +142,10 @@ CommandQueue<Command>::peekNextCommand() const
 
 template<class Command>
 void
-CommandQueue<Command>::add(
-        const std::shared_ptr<Command>& cmd)
+CommandQueue<Command>::add(const std::shared_ptr<Command>& cmd)
 {
     framework::MicroSecTime time(_clock.getTimeInMicros()
-            + framework::MicroSecTime(cmd->getQueueTimeout() * 1000000));
```

Comment:
@vekterli This is where I suspect it adds it to the Q with a timeout that is 1000 X larger than was intended.